data/json/monsters/civilians.json

```
@@ -105,8 +105,12 @@
     "name": "GROUP_CIVILIANS_UPGRADE",
     "type": "monstergroup",
     "monsters": [
-      { "group": "GROUP_VANILLA_ONLY_HUMANS", "weight": 8, "cost_multiplier": 0 },
-      { "group": "GROUP_FERAL", "weight": 2, "cost_multiplier": 0 }
+      { "monster": "mon_zombie", "weight": 16, "cost_multiplier": 0 },
+      { "monster": "mon_zombie_fat", "weight": 16, "cost_multiplier": 0 },
+      { "monster": "mon_zombie_tough", "weight": 16, "cost_multiplier": 0 },
+      { "monster": "mon_zombie_crawler", "weight": 16, "cost_multiplier": 0 },
+      { "monster": "mon_zombie_brainless", "weight": 16, "cost_multiplier": 0 },
+      { "group": "GROUP_FERAL", "weight": 20, "cost_multiplier": 0 }
```

```suggestion
      { "monster": "mon_zombie", "weight": 264, "cost_multiplier": 0 },
      { "monster": "mon_zombie_fat", "weight": 266, "cost_multiplier": 0 },
      { "monster": "mon_zombie_tough", "weight": 100, "cost_multiplier": 0 },
      { "monster": "mon_zombie_crawler", "weight": 66, "cost_multiplier": 0 },
      { "monster": "mon_zombie_brainless", "weight": 30, "cost_multiplier": 0 },
      { "group": "GROUP_FERAL", "weight": 180, "cost_multiplier": 0 }
```
My point is this would mimic how it was more accurately, if you want to change the weights though (idk why fat zed is so common) feel free

---

OHHHH that's what you meant. Okay, that's fair. And yeah, I'll commit this. I completely forgot the initial mongroup had them in different proportions, thanks!